core/lib/spree/event/subscriber.rb

```diff
@@ -26,6 +26,8 @@ def self.included(base)
 
         base.mattr_accessor :event_actions
         base.event_actions = {}
+
+        Spree::Config.subscriber_names << base.name
```

Comment:
What is name here?

Reply:
It's `Module#name`, e.g. `Foo::Bar.name # => "Foo::Bar"`